user.tex

```diff
@@ -38,52 +38,53 @@ \subsection{Web Portal\label{sec:portal}}
+will be provided to allow for more sophisticated data selection, analysis, and creation of added value \textit{User Generated} data products. 
+A screen capture of a mature prototype of the Notebook Aspect is shown in Figure~\ref{fig:JupyterLab}.
 
-The JupyterLab user experience will be nearly identical to working with
+The Notebook Aspect user experience will be nearly identical to working with
 Jupyter notebooks locally, except that computation and analysis will occur
-at resources provided at the LSST Data Access Center.  This is an
-implementation of the “bringing analysis to the data” paradigm: rather
+at resources provided at the LSST Data Access Center.
+This is an
+implementation of the ``bringing analysis to the data'' paradigm: rather
 than imposing the burden of downloading, storing, and processing (large)
 subsets of LSST data at their home institutions, we will enable our users to
-bring their codes and perform their analysis at the LSST DAC.  We expect
+bring their codes to and perform their analysis at the LSST DAC.  We expect
 this will reduce the barrier to entry and shorten the path to science for
 the LSST science community.
 
 We will provide JupyterLab instances to LSST users in an environment carrying a
 library of preinstalled commonly used and useful software tools:
-Astropy, the LSST science pipelines, Anaconda Scientific Python Distribution,
-and others. The users will be able to upload and install their own tools as well.
-Non-trivial shared computing cluster resources will be accessible through this
-environment as well, enabling Level 3 data products generation.
+Astropy, the LSST science pipelines, Anaconda Scientific Python Distribution, the PyViz visualization toolkit, and others. 
+The users will be able to upload and install their own tools as well.
+Non-trivial shared computing cluster resources will be accessible through this environment as well, enabling the generation of \textit{User Generated} data products.
 
 % Add something about writing glue code to make it easy to submit jobs, and how Notebooks will be the primary user interface to the backend batch system...
 
-The JupyterLab aspect of the science platform will play a key role in commissioning,
-quality assessment, and science validation of the as-built system. It will be the primary
-method of performing interactive analysis of acquired data (e.g. adjusting and executing
+The Notebook Aspect of the science platform will play a key role in commissioning,
+quality assessment, and science validation of the as-built system.
+It will be the primary
+method of performing interactive analysis of acquired data (e.g., adjusting and executing
 prepared notebooks driving commissioning tasks), as well as commanding
-the batch resources to execute larger processing tasks. Due to this, we expect the
-JupyterLab aspect to reach maturity earlier than the others, and certainly in time for
+the batch resources to execute larger processing tasks.
```

Comment:
I'll say again here.  I don't believe the baseline is to launch batch jobs from the notebook.  I think we should keep expectations in line with what we are planning on delivering.

Reply:
If we say commanding batch resources through the terminal functionality of the JupyterLab environment, I would be much happier.

Reply:
Sounds good to me.